src/vm/exceptionhandling.cpp

```diff
@@ -17,6 +17,10 @@
 #include "eventtrace.h"
 #include "virtualcallstub.h"
 
+#if defined(_TARGET_X86_) || defined(_TARGET_AMD64_)
```

Comment:
@parjong the overflow is always reported as division by zero on amd64 / x86 Unix. I don't think that this change makes the code clearer, since all the code is in one file close to each other and commented in detail. On the contrary, with your change, someone reading the code has to lookup the definition of the MAY_THROW_DBZ_ON_OVF to find out that it is only related to x86 and amd64, which is the key thing here.
In general, I prefer  to not to create new defines unless the related ifdefs are scattered over multiple places and it is difficult to figure out what is some part of code related to easily. One example of a reasonable usage of a new define was the one you have added for the way of calling funclets.